### PR TITLE
Create migrations section

### PR DESCRIPTION
**Pull request by mcdappdev**
A little addition the documentation section on Preparations that addresses how to use a Preparation to change an existing db schema.

### CODE REVIEW

File: fluent/model.md [hunk 1]
```diff
@@ -123,6 +123,27 @@ final class User {
 
 Here we are deleting the table named `users`.
 
+### Preparations as Migrations
+
+If you want to add a field to your table after you've already created the initial schema, you can create a struct or class that inherits from `Preparation` like so:
```
Comment (LoganWright) [inline fluent/model.md]: `conforms to` not `inherits from`

Comment (mcdappdev) [inline fluent/model.md]: Ah, yes, thank you

File: fluent/model.md [hunk 2]
```diff
@@ -123,6 +123,27 @@ final class User {
...
 
+### Preparations as Migrations
+
+If you want to add a field to your table after you've already created the initial schema, you can create a struct or class that conforms to `Preparation` like so:
+
+```swift
+
+struct AddFooToBar: Preparation {
+    static func prepare(_ database: Database) throws {
+        try database.modify("bars", closure: { bar in
```
Comment (kdawgwilk) [inline fluent/model.md]: Trailing Swift closure syntax I think would be a little more Swifty here

**Comment by kdawgwilk** [comment]
Sorry saw a notification for this and made a comment without realizing it had already merged

**Pull request merged**